### PR TITLE
Feature/135140 hubspot connection

### PR DESCRIPTION
**Pull request by jesseeproductions**
_Ref:_ [C#135140](https://central.tri.be/issues/135140)

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -40,12 +35,53 @@ public static function instance() {
...
+		return $this->settings;
+	}
+
+	/**
+	 * Get all of this extension's options.
+	 *
+	 * @return array
+	 */
+	public function get_all_options() {
+		$settings = $this->get_settings();
```
Comment (sc0ttkclark) [inline src/Tribe/Main.php]:
```suggestion
		$settings = tribe( 'tickets.hubspot.admin.settings' );
```

File: src/Tribe/Main.php [hunk 2]
```diff
@@ -40,12 +35,53 @@ public static function instance() {
...
+	 *
+	 */
+	public function bootstrap() {
+
+		// Intialize the Service Provider for Event Tickets HubSpot Integration.
+		tribe_register_provider( Service_Provider::class );
+
+	}
+
+	/**
```
Comment (sc0ttkclark) [inline src/Tribe/Main.php]: If we call `tribe( 'tickets.hubspot.admin.settings' )` then we don't need this method

File: src/Tribe/Main.php [hunk 3]
```diff
@@ -40,12 +35,53 @@ public static function instance() {
...
+		$this->init();
 
 	}
 
 	public function init() {
 
+		// Setup bootstrap on earliest hook available to the extension.
+		add_action( 'tribe_plugins_loaded', [ $this, 'bootstrap' ], 11 );
+
+		$this->get_settings();
```
Comment (sc0ttkclark) [inline src/Tribe/Main.php]:
```suggestion
```

File: src/Tribe/Main.php (hunk 2 above)
Comment (jesseeproductions) [reply src/Tribe/Main.php]: removed it!

File: src/Tribe/Main.php (hunk 3 above)
Comment (jesseeproductions) [reply src/Tribe/Main.php]: Removed It.

File: src/Tribe/Main.php (hunk 1 above)
Comment (jesseeproductions) [reply src/Tribe/Main.php]: Changed to tribe( 'tickets.hubspot.admin.settings' ).

**Pull request merged**